pkg/v1/tkg/test/tkgctl/shared/e2e_tkrresolver.go

```diff
@@ -236,10 +227,6 @@ func E2ETKRResolverValidationForClusterCRUDSpec(context context.Context, inputGe
 
 		// verify addons are deployed successfully in clusterclass mode
 		if isClusterClass, ok := input.OtherConfigs["clusterclass"]; ok && isClusterClass == "true" {
-			By(fmt.Sprintf("Verify addon packages on management cluster %q matches clusterBootstrap info on management cluster %q", input.E2EConfig.ManagementClusterName, input.E2EConfig.ManagementClusterName))
```

Comment:
question: why can we skip checking addons on the management cluster?

Reply:
Cause the individual tests are only deploying a workload cluster and no need to check addons deployments on the otherwise same management cluster. This check doesn't cause any problems in most test cases, but in the ```aws_cc_custom_cb_test.go```  testcase, which uses custom CB, the following check only holds for workload cluster and would fail for an existing managment cluster: https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/test/tkgctl/shared/package.go#L283

I can add a flag check for ```IsCustomCB``` before doing that test, but will make the code a bit dirty

Reply:
That makes sense, thanks for the explaination! I'm ok with removing the check on management cluster in this case

Reply:
I kept the check as much as possible; Instead, I made changes in [this line](https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/tkg/test/tkgctl/shared/package.go#L283) of code which had not considered management cluster before (as it was originally written with only tkgs in mind maybe)